### DISCUSSION TITLE
personalizzazione

### DISCUSSION THREAD

**Discussion by fabriziotoppi** [iisgiua/giuaschool#532] [Richieste di aiuto]
buonasera come faccio a modiifcare questi 2 campi dalla pagina admin
il campo "codice sorgente du github?
grazie

**Comment by trinko** [comment]
Se intendi il link "codice sorgente" che è visibile sulle pagine del registro, devi modificare il template principale:
- **templates/tema-new/layout-base.html.twig** per le pagine dell'amministratore
- **templates/base.html.twig** per le altre pagine del registro

Segui le indicazioni nella documentazione per le personalizzazione: [Personalizzazione delle pagine](https://iisgiua.github.io/giuaschool-docs/customize.html#personalizzazione-delle-pagine-del-registro)

**Comment by fabriziotoppi** [comment]
Antonello ciao piacere di conoscerti .ti ringrazio  ho installato il
registro e lo sto  personalizzando . Non riesco a configurare la mail in
uscita . Ne tramite SMTP ne tramite Google . Potresti aiutarmi? Grazie

Il gio 15 giu 2023, 18:52 Antonello Dessì ***@***.***> ha
scritto:

**Reply by trinko** [reply]
Vi prego di usare una discussione per ogni problema. Altrimenti è impossibile gestire queste richieste.
Ho risposto nell'altra discussione aperta.

**Comment by ghost** [comment]
@fabriziotoppi Buongiorno! Avrei bisogno di parlarti, in merito a questa richiesta. Posso parlarti in privato? Mi scrivi alla mia mail: padovesi.roberto@gmail.com? Grazie!
